valhalla/baldr/traffictile.h

```diff
@@ -64,15 +64,16 @@ struct TrafficSpeed {
   uint64_t spare : 1;
 
 #ifndef C_ONLY_INTERFACE
-  inline bool valid() const volatile {
-    return breakpoint1 != 0;
+  inline bool speed_valid() const volatile {
+    return breakpoint1 != 0 && overall_speed != UNKNOWN_TRAFFIC_SPEED_RAW;
   }
+
   inline bool closed() const volatile {
-    return valid() && overall_speed == 0;
+    return overall_speed == 0;
```

Comment:
Why was the valid check removed? could speed be 0 without it being explicitly set?

Reply:
It was redundant - I've adjusted, it really needs to consider the `breakpoint1` value.